src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -1,81 +1,126 @@
 // WE ARE CURRENTLY TRYING OUT THE SingleBookingComponent.js PLEASE REFER TO THAT COMPONENT FOR BOOKING FOR NOW
-
 import React from 'react';
 import { connect } from 'react-redux';
-import { Card, Button } from 'antd';
+import { Typography, Input, Select, Layout, Form } from 'antd';
 import { dateConverter } from '../../common/dateHelpers';
 import { timeConverter } from '../../common/timeHelpers';
 import axiosWithAuth from '../../../utils/axiosWithAuth';
 import { useOktaAuth } from '@okta/okta-react';
 import { addToCart } from '../../../redux/actions/parentActions';
+import { dummyData } from '../../../dummyData';
 
 const ParentBookingCard = props => {
   const {
-    child_name,
-    subject,
-    description,
+    course_name,
+    course_description,
```

Comment:
I think at some point we need to address that times could be different on different days. Example: Monday class could start at 9am, but Tuesday, class starts at 3pm.